src/scan_to_cloud_filter_chain.cpp

```diff
@@ -260,18 +280,28 @@ class ScanToCloudFilterChain
 
 } ;
 
+#if BUILDING_NODELET
 
+class ScanToCloudFilterChainNodelet : public nodelet::Nodelet
+{
+  std::unique_ptr<ScanToCloudFilterChain> chain_;
+
+  void onInit() override {
+    chain_ = std::make_unique<ScanToCloudFilterChain>(getPrivateNodeHandle(), getName());
```

Comment:
@peci1 looks like `std::make_unique` is only C++14 onwards. Since the kinetic-devel branch should compile on kinetic and after, could you replace that with a regular `chain_ = std::unique_ptr<ScanToCloudFilterChain>(new ...)`?

I'm planning to make a noetic branch that will take all future changes; another option would be to merge this into that branch.

Reply:
I've made the requested change.